examples/phewas_scatter.html

```diff
@@ -22,40 +22,131 @@
         max-width: 100%;
         box-sizing: border-box;
       }
+      pre {
+        overflow: auto;
+        background-color: #f1f1f1;
+        border-radius: 3px;
+        font-size: 1em;
+      }
+      code {
+        font-size: 1em;
+      }
+      .spacer {
+        padding: 0px 15px 0px 15px;
+      }
+      .required {
+        color: red;
+      }
+      .error {
+        color: red;
+      }
     </style>
 
   </head>
 
   <body>
     <div class="container">
 
-      <h1 style="margin-top: 1em;"><strong>Locuszoom.js</strong></h1>
+      <h1 style="margin-top: 1em;"><strong>LocusZoom.js</strong></h1>
+
+      <h3 style="color: #777">PheWAS Demonstration</h3>
 
-      <h3 style="float: left; color: #777">PheWAS (Scatter) Example</h3>
-      <h6 style="float: right;"><a href="../index.html">&lt; return home</a></h6>
+      <p>
+        A PheWAS visualization displays association p-values between a given genetic variant and multiple phenotypes. 
+        In this demonstration, we are showing single variant association statistics from multiple publicly available meta-analyses of genome-wide association studies (GWAS), stored locally in a database at the Center for Statistical Genetics (University of Michigan.)
```

Comment:
Very small terminology note: the word "locally" usually refers to a local machine. In practice, the data is coming from a remote server. (and when hosted online at `statgen.github.io/locuszoom`, not the same server as the demo page)